### PR TITLE
Issue 274: Make documentation up to date

### PR DESCRIPTION
**Pull request by damien-carcel**
## Description

This PR is a big cleaning and update of the documentation, even if there is still some work to do (see the corresponding issue).

The update is especially important for the Akeneo part of the documentation, which was absolutely not in sync with the last changes in the images.

There is still work to do on this part too, which come later, with the move of this section in the official Akeneo documentation. **The goal here is only to make the Dockerfiles documentation right again, not to make it as complete as possible!**

### Side note

Two files `docker-compose.mac.yml.dist` and `docker-sync.yml.dist` where moved in a separate folder. They are a recent contribution from @torhoehn, that are in fact applicable to Symfony in general, not just Akeneo, and they will need a dedicated documentation to explain their usage, in a dedicated PR. 

## Definition Of Done

| Q                                 | A
| --------------------------------- | ---
| Added tests                       | -
| Changelog updated                 | -
| Documentation                     | -
| Fixed issues                      | #274 

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Contributing
 
-These Dockerfiles are not only made for my personal use, but for anybody who likes to develop on Akeneo or Symfony applications with Docker.
+These Dockerfiles are for anybody who likes to develop on Akeneo or Symfony applications with Docker.
 
-I gladly accept any contributions: fixing typos, glitches, bugs, technical enhancements and new features.
+We gladly accept any contributions: fixing typos, glitches, bugs, technical enhancements and new features.
 
-For technical enhancements and new features, please first have a look at [current issues](https://github.com/akeneo/Dockerfiles/issues).
+For technical enhancements and new features, please first have a look to the [current issues](https://github.com/akeneo/Dockerfiles/issues).
```
Comment (tamarasaurus) [inline CONTRIBUTING.md]: at sounds better 😛

File: Docs/akeneo/compose.md [hunk 2]
```diff
@@ -1,18 +1,21 @@
 # Akeneo development and testing with docker-compose
 
 **IMPORTANT - These images do not contain Akeneo.** 
-They are intended to be used with an already existing (or new) Akeneo project on your host machine, that will be mount as a data volume.
+They are intended to be used with an already existing (or new) Akeneo project on your host machine, that will be mounted as a data volume.
 
-Two compose file examples are provided, both providing two environments, one for development, the other for running behat and integration tests:
+Two compose file examples are already included in `akeneo/pim-community-dev` and `akeneo/pim-enterprise-dev` starting with version 1.7.
+Both provides two environments, one for development, the other for running tests (functional tests with behat and integration tests with PHP Unit):
```
Comment (tamarasaurus) [inline Docs/akeneo/compose.md]: provide

File: Docs/akeneo/fpm.md [hunk 3]
```diff
@@ -39,7 +43,10 @@ This, of course, will not delete the Akeneo application you cloned on your machi
 
 ### Configure Akeneo
 
-First, make sure that Akeneo database settings are as the containers expect them:
+First, make sure that Akeneo database settings are as the containers expect them.
```
Comment (tamarasaurus) [inline Docs/akeneo/fpm.md]: you can remove 'them'

File: Docs/akeneo/fpm.md [hunk 4]
```diff
@@ -39,7 +43,10 @@ This, of course, will not delete the Akeneo application you cloned on your machi
...
+As you can see below, the `database_host` is the name of your MySQL service in the compose file.
+For Elasticsearch, `index_hosts` is the association of the login and password (`elastic` and `changeme`, respectively) of the container,
+the service name in the compose file (`elasticsearch`) and and the output port of Elasticsearch (`9200`).
```
Comment (tamarasaurus) [inline Docs/akeneo/fpm.md]: double and

File: Docs/akeneo/fpm.md [hunk 5]
```diff
@@ -68,76 +75,49 @@ parameters:
...
-- [`pim-initialize` script](https://github.com/akeneo/Dockerfiles/blob/master/akeneo-fpm/files/pim-initialize.sh) will completely remove the application cache, logs, assets (`web/bundles/*`, `web/css/*`, `web/js/*`),
-    and everything generated by any previous installation. Then it will perform a installation of Akeneo in production mode through the command `pim:install --force`, and dump all assets as symlinks.
-    Finally, it will initialize the database (through `pim:installer:db`) for the behat environment.
-
-    Basically, this command will provide you with a fresh PIM, as if it was its first installation.
-
 ## Run behat tests
 
-The tests are to be run inside the containers. Start by configuring Behat:
+The tests are to be run inside the containers. Start by configuring Behat as follow:
```
Comment (tamarasaurus) [inline Docs/akeneo/fpm.md]: as follows

File: Docs/getting-started.md [hunk 6]
```diff
@@ -25,26 +25,26 @@ The official documentation proposes to use Docker own repositories and packages,
...
 
-The best way to understand a new tool is, in my opinion, an example. You can see below a very basic compose file.
+As the best way to understand a new tool is an example, you can see below a very basic compose file.
 
 It allows to run a Symfony application (but it would work with other PHP web application too) with Apache, PHP and MySQL.
 
 There is several versions of the compose file format. The most recent it is, the most recent version of `Docker` and `Docker Compose` you need to run it.
-I chose version 2.0 because it has a lot of improvements regarding the 1.0, and because other versions did not exists when I started this repository.
-My examples could use more recent version in the future, and I encourage you to look at the new versions, as they could introduce new features that are usefull to you.
+We chose version 2.0 because it has a lot of improvements regarding the 1.0, and because other versions did not exists when we started this repository.
```
Comment (tamarasaurus) [inline Docs/getting-started.md]: did not exist

File: Docs/getting-started.md [hunk 7]
```diff
@@ -25,26 +25,26 @@ The official documentation proposes to use Docker own repositories and packages,
...
+Those examples could use a more recent version in the future, and we encourage you to look at the new versions, as they could introduce new features that are usefull to you.
```
Comment (tamarasaurus) [inline Docs/getting-started.md]: useful

**Review by tamarasaurus — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (damien-carcel) [reply CONTRIBUTING.md]: Done.
